docker-compose.yaml

```diff
@@ -31,7 +31,8 @@ services:
       DEFRA_ID_ENABLED: ${DEFRA_ID_ENABLED:-false}
       DEFRA_ID_TENANT: ${DEFRA_ID_TENANT:-azdcuspoc5}
       DEFRA_ID_POLICY: ${DEFRA_ID_POLICY:-B2C_1A_SIGNUPSIGNINSFI}
-      DEFRA_ID_REDIRECT_URI: ${DEFRA_ID_REDIRECT_URI:-http://localhost:3000/apply/signin-oidc}
+      DEFRA_ID_REDIRECT_URI: ${DEFRA_ID_REDIRECT_URI:-http://localhost:3004/claim/signin-oidc}
       DEFRA_ID_CLIENT_ID: ${DEFRA_ID_CLIENT_ID:-83d2b160-74ce-4356-9709-3f8da7868e35}
```

Comment:
Can you please remove default setting, looks they are real value. it will be exposed in public.

Reply:
I think these are fine as they are not strictly secrets. They will be in the ouath authorize URL that customers are redirected to and visible in the browser